### PR TITLE
[DOCU-2514] Konnect latency analytics

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
Adding docs for the new latency data in Konnect.
* Description and screenshot updates for all analytics dashboard images
* Updates to vitals intro page and runtime groups dashboard doc
   
   **Why did I remove one of the screenshots on the Analytics intro page?** Because we should have either all views, or only one sample tab screenshot on that page. It was turning into a clone of the summary dashboard page, and it was hard to tell the difference between the two - so I opted to include one sample screenshot on the overview, then all screenshots on the dashboard doc.

* Troubleshooting topic for what to do if you see no data in the latency tab. I couldn't think of other common Analytics issues, but decided to add a separate topic to follow the pattern we use for other Konnect sections.

### Reason
DOCU-2514

I'd also like to follow up and revisit our approach for documenting analytics, after this is out. At the moment, we're just documenting the UI. This should instead be a reference that covers metrics available throughout the product.

### Testing
[Release note](https://deploy-preview-4630--kongdocs.netlify.app/konnect/updates/)
[Analytics intro](https://deploy-preview-4630--kongdocs.netlify.app/konnect/analytics/)
[Analytics dashboard](https://deploy-preview-4630--kongdocs.netlify.app/konnect/analytics/summary-dashboard/)
[Runtime groups dashboard](https://deploy-preview-4630--kongdocs.netlify.app/konnect/runtime-manager/runtime-groups/dashboard/)
[Troubleshoot analytics](https://deploy-preview-4630--kongdocs.netlify.app/konnect/analytics/troubleshoot/)

### CODE REVIEW

File: app/konnect/analytics/summary-dashboard.md [hunk 1]
```diff
@@ -9,24 +9,34 @@ The Summary Dashboard provides metrics for services cataloged by Service Hub wit
 
 * **Traffic**: Total number of HTTP requests.
 * **Errors**: Percentage of failed HTTP requests.
+* **Latency**: Request latency at P90, P95, and P99 percentiles.
```
Comment (sichvoge) [inline app/konnect/analytics/summary-dashboard.md:12]: Should be "P50, P95, and P99".

File: app/konnect/analytics/summary-dashboard.md [hunk 2]
```diff
@@ -9,24 +9,34 @@ The Summary Dashboard provides metrics for services cataloged by Service Hub wit
...
+    ![traffic analytics graph](/assets/images/docs/konnect/konnect-vitals-traffic.png)
     > _**Figure 1:** Graph showing successful and failed requests over the past three hours._
 
 * **Errors**: This graph displays the total number of failed HTTP requests categorized by error response codes over the specified time interval. Error response codes include any **4xx-5xx** status codes.
 
-    ![errors vitals graph](/assets/images/docs/konnect/konnect-vitals-errors.png)
+    ![errors analytics graph](/assets/images/docs/konnect/konnect-vitals-errors.png)
     > _**Figure 2:** Graph showing errors by 4xx and 5xx error codes received over the past three hours._
 
+* **Latency**: This graph displays request latency, in milliseconds, of the 99th, 95th, and 90th percentiles.
```
Comment (sichvoge) [inline app/konnect/analytics/summary-dashboard.md:31]: Should be "99th, 95th, and 90th"

**Review by sichvoge — CHANGES_REQUESTED** [review]
Thajnks @lena-larionova. There are only two changes and then we are good to go!

File: app/konnect/analytics/summary-dashboard.md (hunk 2 above)
Comment (mheap) [reply app/konnect/analytics/summary-dashboard.md:31]: Did you mean 99th, 95th and 50th?

Comment (sichvoge) [reply app/konnect/analytics/summary-dashboard.md:31]: Yes. Sorry.

**Review by mheap — APPROVED** [review]

**Pull request merged**